core/tests/protractor/collections.js

```diff
@@ -30,7 +31,6 @@ describe('Collections', function() {
       '.protractor-test-reload-collection-button')).first().click();
     general.acceptAlert();
     browser.waitForAngular();
-    admin.reloadAllExplorations();
```

Comment:
That's a bit odd -- I'd have thought that GitHub would handle the merge automatically. Can you try merging upstream/develop into your branch and pushing again?